package.json

```diff
@@ -2,40 +2,46 @@
   "name": "proxy-live-document",
   "version": "1.0.0",
   "description": "Mutable, observable, patchable JSON state wrapped in Smart Domains",
-  "types": "./dist/index.d.ts",
-  "typings": "./dist",
-  "main": "./dist/index.main.js",
-  "module": "./dist/index.module.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vladnicula/proxy-live-document"
+  "types": "dist/index.d.ts",
+  "main": "dist/index.main.js",
+  "module": "dist/index.module.js",
+  "browser": "dist/index.browser.js",
```

Comment:
I had to remove this one previously because the browser bundle version did not work with nextjs apps. Did you test the repo in playgorund maybe and confired this bundle works?

Reply:
Nope, i did not check in SSR based apps, i will check 👍

Reply:
Hi vlad, here is a JSFiddle that i created with the browser bundle --> https://jsfiddle.net/jkrishna95/esv0x9n5/7/

It works fine, i think that problem you had with NextJS is because. This should be loaded after rendering the app and making window available.

Reply:
ok, so it seems the library is not working in nextjs. Because we are using it with `import` not `window.SOME_GLOBAL` the methods `select`, `mutate` and the others are not working. 

I'll see if I can remove this browser target from the build and merge your MR without it.

Reply:
Well, looks like I can't edit the code as it's on your repo. Could you please comment out / remove the browser target for now (both from bundles and from package json)

Reply:
Done